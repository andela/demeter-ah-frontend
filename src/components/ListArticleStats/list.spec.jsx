import React from 'react';
import { shallow } from 'enzyme';
import List from './list';

describe('List Component', () => {
  it('Should render without errors', () => {
    const articles = [];
    const component = shallow(<List articles={articles} />);
    expect(component).toMatchSnapshot();
  });
});
