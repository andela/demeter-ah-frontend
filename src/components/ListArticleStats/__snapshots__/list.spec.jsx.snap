// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List Component Should render without errors 1`] = `
<tr
  className="text-center"
>
  <td
    className="p-3 px-5 text-center"
  />
  <td
    className="p-3 px-5 text-center"
  />
  <td
    className="p-3 px-5 text-center"
  >
    NO Available stats
  </td>
  <td
    className="p-3 px-5 text-center"
  />
  <td
    className="p-3 px-5 text-center"
  />
</tr>
`;
